DiceKeys/Views/DerivationRecipes/DerivationRecipeBuilders.swift

```diff
@@ -113,10 +142,13 @@ struct DerivationRecipeForFromUrl: View {
                 textfield.keyboardType(.alphabet)
```

Comment:
Since there are now multiple types of text fields, maybe rename to urlTextField?